kaavapino/urls.py

```diff
@@ -1,30 +1,11 @@
 from django.conf import settings
 from django.conf.urls.static import static
 from django.contrib import admin
-from django.contrib.auth import views as auth_views
-from django.urls import include, path
-from django.views.generic import TemplateView
-from stronghold.views import StrongholdPublicMixin
-
-from projects import views as projects_views
+from django.urls import path
 
 admin.autodiscover()
 
-
-class PublicTemplateView(StrongholdPublicMixin, TemplateView):
-    """TemplateView but with django stronghold public mixin"""
-
-
-urlpatterns = [
-    path('', PublicTemplateView.as_view(template_name='index.html')),
-    path('accounts/', include('allauth.urls')),
-    path('admin/', admin.site.urls),
-    path('login/', auth_views.LoginView.as_view(template_name='login.html'), name='login'),
```

Comment:
Should we leave the login views in place? :thinking: I guess they might be needed for going into the Django admin?

Reply:
I check, we don't need them.